addons/web/static/src/webclient/actions/action_service.js

```diff
@@ -946,11 +946,12 @@ function makeActionManager(env) {
         }
 
         if (env.isSmall) {
-            if (!view.isMobileFriendly) {
-                view = _findMobileView(views, view.multiRecord) || view;
+            const has_mobile_mode = action.default_mobile_mode && views.map((v) => v.type).includes(action.default_mobile_mode)
```

Comment:
we camelcase variables in js.

Also, there's a missing `;` here. You could/should enable  the linting via ./addons/web/tooling/enable.sh (it's a pre-commit hook, so you'll have to re-do the commit)

Reply:
This line is actually not necessary as `_findMobileView` won't return anything if there's no mobile view